fastlane/Fastfile

```diff
@@ -406,23 +406,27 @@ lane :update_docs do |options|
   end
 end
 
-def clone_docs(debug: false)
+def clone_docs(debug: false, target_dir: nil)
```

Comment:
Instead of making this a parameter that gets passed in, can we make this an environment variable called `FASTLANE_CLONE_PATH`? This removes the hardcoding of `/tmp/validated_docs` on line 319 which won't work on Windows? We can then set this path dynamically on any CI system

Reply:
Would have to be `FASTLANE_DOCS_CLONE_PATH` as this refers to the docs I think. But yep, sounds useful. Will make the change.